lib/engine/game/g_18_eus/round/auction.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require_relative 'bidbox_auction'
+
+module Engine
+  module Game
+    module G18EUS
+      module Round
+        class Auction < Engine::Round::Auction
+          include Engine::Game::G18EUS::Round::BidboxAuction
+
+          attr_accessor :bids
+
+          def setup
+            @taken_loans = []
+            @paid_loans = []
```

Comment:
These aren't used here. Move them to where they are used.

Reply:
Removed - not needed at all here